src/course/syllabus/application/website/learning-outcomes.md

```diff
@@ -0,0 +1,58 @@
+## Planning
+
+- [ ] Be confident in planning layout, styling, and featurs of a a website
+- [ ] Be able to plan a workflow for building a website
+- [ ] Be able to break a website into distinct sections to build
+
+## HTML
+
+- [ ] Be confident in using HTML tags to build a website
+- [ ] Use semantic HTML tags on a website
+
+## CSS
+
+- [ ] Be confident in applying styles using CSS
+- [ ] Be able to organise a stylesheet
+- [ ] Be confident in sizing elements in CSS
+- [ ] Be able to use CSS to achieve a consistent design across a site
+- [ ] Be able to use media queries to build a responsive website
+- [ ] Understand and be able to implement the CSS Box Model on a website
```

Comment:
I'm not sure about having css and html learning outcomes here - they have been covered earlier on and feel like an unnecessary repeat? The planning section feels more relevant as an  immediate learning outcome for this project.